File: 0169-majority-element/0169-majority-element.cs
public class Solution {
    public int MajorityElement(int[] nums) {
        int count =0;
        int current=0;

        for(int i =0;i<nums.Length;i++){
             if(count ==0){
                current = nums[i];
            }
            if(current == nums[i]){
                count++;
            }
            else{
                count --;
            }
            if(count ==0){
                current = nums[i];
            }

        }
        return current;
    }
}

File: 0169-majority-element/0169-majority-element.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = {}
        for n in nums:
            count[n] = count.get(n, 0) + 1
            if count[n] > len(nums) // 2:
                return n

            
                
            

        